### DISCUSSION TITLE
Cannot search in service list

### DISCUSSION THREAD

**Discussion by luojie233** [apache/skywalking#8391] [Q&A]
Question: I cannot search a service in this list, the list shows 26 services, but I can't find the one i want. The service api info or other event info can show in UI normally except the service list

![image](https://user-images.githubusercontent.com/17927730/148196532-070858bd-ec9c-4b34-a1b8-a1d2da9a1e13.png)

Environment:
skywalking-oap: 8.9.1
skywalking-ui: 8.9.1
skywalking-agent: 8.8.0

**Comment by wu-sheng** [comment]
Sorry, your provided information is very limited.

**Reply by luojie233** [reply]
I found 26 services in services query yesterday, but now is just 15, i don't know what happened.
![image](https://user-images.githubusercontent.com/17927730/148315587-2c3a1030-9428-4fc2-8417-a270d7379534.png)

![image](https://user-images.githubusercontent.com/17927730/148315652-cc6639a7-86e9-4408-9ad7-ba92280b90aa.png)

**Reply by luojie233** [reply]
i restarted one oap server (i have two server), and it recovered

**Reply by wu-sheng** [reply]
Never faced a case like this. Sorry.
